src/styles/blocks/button.scss

```diff
@@ -0,0 +1,18 @@
+.button {
+  justify-content: center;
+  align-items: center;
+
+  display: flex;
+
+  transition: background-color $animation-duration ease-in-out;
+
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #fff;
```

Comment:
if you use variables for colors, use them for all the colors

Reply:
Thanks a lot for the review👌 And speaking of input phone validation, let me guess, you didn't get it right for the first time, did you?😂 I'm going to leave it as it is, it might be too rigid, I know. I just think it's better to receive less numbers that are right than more broken ones

Reply:
Well, I've never seen such phone number validation anywhere on real websites, when it's just a form for a customer who wants you to call him back. It's not user friendly, and there is basically no reason for such validation, as if you just limit number of chars and their type, only valid number will work anyway. Your validation is not about detecting invalid numbers, but about following strict pattern, which is not required anywhere on modern websites.